assessments/guide/project-lead.md

```diff
@@ -1,6 +1,6 @@
 # Project Lead
 
-In the context of the security assessment, the "project lead" should be
+In the context of the project security assessment, the "project lead" should be
```

Comment:
see above comments.  Let's pick something and make it consistent.  "project security assessment" only makes sense if there's another kind of security assessment.  

cc @JustinCappos in case he has thoughts on naming this thing :)

Reply:
made adjustments per previous